lib/hz2600/Kazoo/Api/Entity/Conference.php

```diff
@@ -38,4 +38,22 @@ public function executeDial($envelope) {
 
         return $endpoints;
     }
+    
+    public function participantAction($id, $action) {
+        if (!is_numeric($id)) {
+            throw new RuntimException("Invalid participant ID");
```

Comment:
Also I think you need to use this exception for it to actually throw. 

```suggestion
            throw new RunTimeException("Invalid participant ID");
```

Reply:
Yes, it was a typo, thanks.